cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -329,11 +329,12 @@ define(["jquery", "js/common_helpers/ajax_helpers", "js/views/utils/view_utils",
                     var reindexSpy = spyOn(outlinePage, 'startReIndex').andCallThrough();
                     var successSpy = spyOn(outlinePage, 'onIndexSuccess').andCallThrough();
                     var reindexButton = outlinePage.$('.button.button-reindex');
-                    var test_url = '/course_search_index/5';
+                    var test_url = '/course/5/search_reindex';
                     reindexButton.attr('href', test_url)
+                    var fakeserver = Sinon.fakeServer.create();
```

Comment:
Hmmm... I see that it was using the helpers before. If there is a requirement that they don't meet I'd rather see them updated than to start using Sinon.